pkgs/os-specific/linux/kernel/generic.nix

```diff
@@ -222,23 +222,69 @@ let
         buildPhase = ''
           export buildRoot="''${buildRoot:-build}"
 
+          export HOSTCC=${lib.getExe' buildPackages.stdenv.cc "${buildPackages.stdenv.cc.targetPrefix}cc"}
```

Comment:
This needs to be lifted somewhere, and reused. We can't have this in two places at once and risk getting out of sync.

Reply:
Where should that be?

Reply:
I don't know, but it should be in _one_ place and not two.

Reply:
This is done now.

Reply:
Yeah, I like this a lot more now.